crates/mipsy_lib/src/compile/checker.rs

```diff
@@ -58,24 +57,8 @@ pub fn check_post_data_label(program: &MpProgram, binary: &Binary) -> MipsyResul
         match item {
             MpItem::Instruction(ref instruction) => {
                 for (argument, col, col_end) in instruction.arguments() {
-                    match argument {
-                        MpArgument::Register(_) => {}
-                        MpArgument::Number(number) => match number {
-                            MpNumber::Immediate(imm) => {
-                                check_imm(binary, imm, file_tag.clone(), line, *col, *col_end)?
-                            }
-                            MpNumber::Constant(cnst) => check_imm(
-                                binary,
-                                &eval_constant(binary, cnst, file_tag.clone())?.into(),
-                                file_tag.clone(),
-                                line,
-                                *col,
-                                *col_end,
-                            )?,
```

Comment:
this might still be good to do here actually since although the `check_imm` doesnt do anything, propagating the `eval_constant` errors might still be useful